Samples/ResourceManagement/cpp-console-unpackaged/console_unpackaged_app.vcxproj

```diff
@@ -34,9 +34,11 @@
     <VCProjectVersion>16.0</VCProjectVersion>
     <Keyword>Win32Proj</Keyword>
     <ProjectGuid>{fd5c9f04-43c5-4a1c-b3f2-f6725a4ca4ac}</ProjectGuid>
+    <WindowsPackageType>None</WindowsPackageType>
     <RootNamespace>consoleunpackagedapp</RootNamespace>
     <WindowsTargetPlatformVersion>10.0.19041.0</WindowsTargetPlatformVersion>
     <WindowsTargetPlatformMinVersion>10.0.17763.0</WindowsTargetPlatformMinVersion>
+    <LogSDKReferenceResolutionErrorsAsWarnings>true</LogSDKReferenceResolutionErrorsAsWarnings>
```

Comment:
is this addition needed for update to 1.0?

Reply:
Removing this generates WAS known errors for which this is an also known workaround. I know this has been used across several Microsoft projects such as WinUI.